cmd/osm-controller/osm-controller.go

```diff
@@ -284,15 +254,16 @@ func main() {
 	log.Info().Msgf("Stopping osm-controller %s; %s; %s", version.Version, version.GitCommit, version.BuildDate)
 }
 
+// getHTTPHealthProbes returns the HTTP health probes served by OSM controller
 func getHTTPHealthProbes() []health.HTTPProbe {
-	return []health.HTTPProbe{
-		{
-			// HTTP probe on the sidecar injector webhook's port
-			URL: joinURL(fmt.Sprintf("https://%s:%d", constants.LocalhostIPAddress, injectorConfig.ListenPort),
-				injector.WebhookHealthPath),
-			Protocol: health.ProtocolHTTPS,
-		},
-	}
+	// Example:
```

Comment:
Should we bring this back to life?

Reply:
This is meant to be removed since the local HTTP probe from the controller to webhook port is no longer necessary due to the webhook port being probed on the `osm-injector` service directly.

What you see in comments is just an example of what the function would return if there were http probes as before.